app/src/main/java/com/khmelenko/lab/varis/log/TravisCommandLeaf.java

```diff
@@ -0,0 +1,9 @@
+package com.khmelenko.lab.varis.log;
+
+public class TravisCommandLeaf extends LogEntryComponent {
+    public String command;
```

Comment:
I would prefer to access this field via getter

Reply:
Sure